extensions/itunes/src/album.tsx

```diff
@@ -0,0 +1,253 @@
+import {
+  ActionPanel,
+  List,
+  showToast,
+  ToastStyle,
+  randomId,
+  popToRoot,
+  closeMainWindow,
+  Detail,
+  Icon,
+  PushAction,
+  getPreferenceValues,
+} from "@raycast/api";
+import { useState, useEffect, useRef, FC } from "react";
+import { AbortError } from "node-fetch";
+import fs from "fs";
+import readline from "readline";
+import { execSync } from "child_process";
+import os from "os";
+
+interface Preferences {
+  path: string;
+}
+
+interface SearchState {
+  results: SearchResult[];
+  isLoading: boolean;
+}
+
+interface SearchResult {
+  id: string;
+  artist: string;
+  album: string;
+}
+
+interface runScriptProps {
+  album: string;
+  artist: string;
+}
+
+interface searchEntry {
+  artist: string;
+  album: string;
+}
+
+let match: searchEntry[] = [];
+const preferences: Preferences = getPreferenceValues();
+const using = "iTunes";
+const XMLPath = preferences.path.replace(/^\s/, "").replace(/^~/, os.homedir());
+if (!fs.existsSync(XMLPath)) {
+  showToast(
+    ToastStyle.Failure,
+    "Library XML File Not Found",
+    "Please correct the XML file path in extension preferences."
+  ).then();
+  throw "Invalid XML file path";
+}
```

Comment:
Instead of throwing an error which is not really user friendly, you could do this check in your `Command` component and if it fails, display a `Detail` with some instructions about how to fix it